src/java/us/kbase/templates/module_travis.vm.properties

```diff
@@ -1,6 +1,31 @@
-language: 
+language: $language
+
+sudo: required
+
+services:
+  - docker
+
+branches:
+  only:
+    - master
+
 env:
+
 before_install:
+  - docker version
+
 install:
+  - pushd ..
+  - git clone https://github.com/kbase/jars
+  - git clone https://github.com/kbase/kb_sdk
+  - cd kb_sdk
+  - make bin
+  - make sdkbase
```

Comment:
This is going to be really slow... not sure if we want to wait until we have a slimmer base image